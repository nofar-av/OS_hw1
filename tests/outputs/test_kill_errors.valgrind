==28767== Memcheck, a memory error detector
==28767== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28767== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28767== Command: /home/omer/OS_wet1/smash
==28767== Parent PID: 28763
==28767== 
==28767== 
==28767== FILE DESCRIPTORS: 4 open at exit.
==28767== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_kill_errors.valgrind
==28767==    <inherited from parent>
==28767== 
==28767== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_kill_errors.err
==28767==    <inherited from parent>
==28767== 
==28767== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_kill_errors.out
==28767==    <inherited from parent>
==28767== 
==28767== Open file descriptor 0:
==28767==    <inherited from parent>
==28767== 
==28767== 
==28767== HEAP SUMMARY:
==28767==     in use at exit: 0 bytes in 0 blocks
==28767==   total heap usage: 80 allocs, 80 frees, 86,844 bytes allocated
==28767== 
==28767== All heap blocks were freed -- no leaks are possible
==28767== 
==28767== For counts of detected and suppressed errors, rerun with: -v
==28767== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
