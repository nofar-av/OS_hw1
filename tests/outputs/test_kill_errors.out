smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash: sending SIGKILL signal to 1 jobs:
5809: sleep 4&
